src/gc/gcinterface.h

```diff
@@ -526,6 +527,9 @@ class IGCHeap {
     // throughout the VM.
     virtual HRESULT GarbageCollect(int generation = -1, BOOL low_memory_p = FALSE, int mode = collection_blocking) = 0;
 
+    // Gets the smallest GC generation that runs finalizers.
+    virtual unsigned GetMinFinalizingGeneration() = 0;
```

Comment:
Hmm somehow I was under the other impression, looks like gen 0 is fine. I'll fix to specify 1 or max based on the other discussion. Doing only gen 0 GC probably would not cover enough cases.